samples/python/evaluate_qrcode/evaluate_qrcode.py

```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/env python
+
+'''
+
+This sample uses dataset from
+http://www.fit.vutbr.cz/research/groups/graph/pclines/pub_page.php?id=2012-SCCG-QRtiles
+and markup files located in markup/ folder
+for evaluation QR-code detection algorithm
+
+'''
+
+# Python 2/3 compatibility
+import optparse
+import os
+import sys
+from os.path import join
```

Comment:
please use pathlib instead os.path, all reasons mentioned [this](https://habr.com/ru/post/453862/)

Reply:
thanks! fixed it